qp_qiime2/__init__.py

```diff
@@ -21,176 +25,171 @@
 
 # Define the rarefy command
 req_params = {
-    'i-table': ('artifact', ['BIOM']),
-    'p-sampling-depth': ['integer', 1000]
+    'BIOM table': ('artifact', ['BIOM']),
+    'Sampling depth': ['integer', 1000]
 }
 opt_params = {}
 outputs = {'o-table': 'BIOM'}
 dflt_param_set = {
     'Defaults': {
-        'p-sampling-depth': 1000}
+        'Sampling depth': 1000}
 }
 qiime_cmd = QiitaCommand(
-    "Rarefy", "Rarefy",
+    "Rarefy features", "Rarefy",
     rarefy, req_params, opt_params, outputs, dflt_param_set,
     analysis_only=True)
 plugin.register_command(qiime_cmd)
 
 # Define the beta_diversity command
-req_params = {'i-table': ('artifact', ['BIOM'])}
+req_params = {'BIOM table': ('artifact', ['BIOM'])}
 opt_params = {
-    'p-metric': [
-        ('choice:["sokalsneath", "kulsinski", "mahalanobis", "matching", '
-         '"euclidean", "correlation", "yule", "russellrao", "hamming", '
-         '"jaccard", "braycurtis", "dice", "rogerstanimoto", "sqeuclidean", '
-         '"cityblock", "sokalmichener", "cosine", "wminkowski", "seuclidean", '
-         '"chebyshev", "canberra", "unweighted UniFrac", '
-         '"weighted normalized UniFrac", "weighted unnormalized UniFrac"]'),
-        'jaccard'],
-    'i-tree': ['choice:["default", "None"]', 'None']}
+    'Diversity metric': [
+        'choice:%s' % dumps(list(BETA_DIVERSITY_METRICS.keys())),
+        'Jaccard similarity index'],
+    'Phylogenetic tree': ['choice:["default", "None"]', 'None']}
 outputs = {'distance_matrix': 'distance_matrix'}
 dflt_param_set = {
     'Defaults': {
-        'p-metric': 'jaccard',
-        'i-tree': 'None'}
+        'Diversity metric': 'Jaccard similarity index',
+        'Phylogenetic tree': 'None'}
 }
 qiime_cmd = QiitaCommand(
-    "beta_diversity", "Beta Diversity",
+    "Calculate beta diversity", "Beta Diversity",
     beta_diversity, req_params, opt_params, outputs, dflt_param_set,
     analysis_only=True)
 plugin.register_command(qiime_cmd)
 
 # Define the pcoa command
-req_params = {'i-distance-matrix': ('artifact', ['distance_matrix'])}
+req_params = {'Distance matrix': ('artifact', ['distance_matrix'])}
 opt_params = {}
 outputs = {'o-pcoa': 'ordination_results'}
 dflt_param_set = {
     'Defaults': {}
 }
 qiime_cmd = QiitaCommand(
-    "pcoa", "Principal Coordinate Analysis",
+    "Generate principal coordinates analysis (PCoA)",
```

Comment:
"generate" -> "perform"

Reply:
would it be psosible for the capitalizations of "principal coordinates analysis" to be consistent?